bind/utils.go

```diff
@@ -158,3 +158,16 @@ func getGoVersion(version string) (int64, int64, error) {
 	minor, _ := strconv.ParseInt(version_info[1], 10, 0)
 	return major, minor, nil
 }
+
+func extractPythonName(gname, gdoc string) (string, string) {
```

Comment:
could you add a few table-based tests for this function?
something like:
```go
funcTestExtractPythonName(t*testing.T) {
    for _, tc := range []struct{ doc, wantName, wantDoc string } { ... } {
         ...
    }
}
```

also, it would be great to present the user with a nice error message (instead of some (cryptic) compilation error?) when there's a syntax error in the directive. like:
```go
//gopy:name invalid name_with_a_space
func Foo() { ... }
```

Reply:
I added a simple check. A more complete one should test that the identifier is valid C/Go/Python, but this should be done for all emitted strings in general, I think. (One day:)

Reply:
The code still just panics on error. But perhaps it keeps the code shorter and more readable than propagating the errors all the way up.